src/main/features/dashboard/views/macro/claimStatus/claimPartAdmissionClaimant.njk

```diff
@@ -13,18 +13,24 @@
 
 {% macro claimPartAdmittedAlreadyPaidForClaimantClaimDetails(claim) %}
   <div class="status-block">
-    <h2 class="heading-small">{{ t('The defendant’s response') }}</h2>
+    <h2 class="heading-small">{{ t('Respond to the defendant') }}</h2>
     <p>{{ t('{{ defendantName }} says they paid you {{ amount }} on {{ paymentDate }}.',
         { defendantName: claim.claimData.defendant.name, amount: claim.response.amount | numeral, paymentDate: claim.response.paymentDeclaration.paidDate | date }) }}</p>
     <p>{{ t('You can accept or reject this response.') }}</p>
-    <p>{{ t('You need to respond before 4pm on {{ respondDate }} or your claim won’t proceed.', { respondDate: claim.respondToResponseDeadline | date }) }}</p>
     {{ viewAndRespondOffer(claim, 'Respond') }}
   </div>
 {% endmacro %}
 
 {% macro claimPartAdmittedToOweForClaimantDashboard(claim) %}
-  {{ t('The defendant believes they owe you {{ amount }}. You can accept or reject that this is the amount owed.',
-    { amount: claim.response.amount | numeral }) }}
+
+  {% if claim.response.paymentIntention.paymentOption === domain.PaymentOption.IMMEDIATELY
+     or claim.response.paymentIntention.paymentOption === domain.PaymentOption.BY_SPECIFIED_DATE
+     or claim.response.paymentIntention.paymentOption === domain.PaymentOption.INSTALMENTS %}
```

Comment:
This looks like you just need to know if the `paymentOption` field is set, in which case the three lines can be abbreviated to `{% if claim.response.paymentIntention.paymentOption %}`